src/pytom_tm/io.py

```diff
@@ -68,6 +68,12 @@ def __call__(self, parser, namespace, values: Union[list[str, str], str], option
             parser.error("{0} can only take one or two arguments".format(option_string))
 
 
+class UnequalSpacingError(Exception):
+    def __init__(self, message):
+        # Call the base class constructor with the parameters it needs
+        super().__init__(message)
```

Comment:
is this needed? could you not just replace this with:
```suggestion
    pass
```

Reply:
you are right, that also works